code/modules/mob/mob.dm

```diff
@@ -1489,7 +1489,7 @@ var/list/slot_equipment_priority = list( \
 		canmove = has_limbs
 
 	if(lying)
-		if(ishuman(src))
+		if(ishuman(src) || ismonkey(src))
```

Comment:
Yep, don't want them to regard slimes (or worse, other player-controlled aliens) as valid prey.